includes/definitions/discovery/primekey-sw.yaml

```diff
@@ -0,0 +1,295 @@
+mib: PK-SOFTWARE-APPLIANCE-V2
+modules:
+    os:
+        version: PK-SOFTWARE-APPLIANCE-V2::pk-SAV2-systemVersion.0
+    sensors:
+        count:
```

Comment:
This shouldn't be a count sensor, it should be storage.

Reply:
I'm not certain that it makes sense to implement this as a storage plugin, as this isn't measuring a filesystem.

I'm not entirely sure what it *is* measuring - I'm assuming there's some mechanism that limits the database to a fixed, possibly expandable file size, but it's not documented.

Happy either way though, it'd only be a few lines to convert it.